src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/input/Microsoft.Android.Sdk.BundledVersions.in.targets

```diff
@@ -0,0 +1,34 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.BundledVersions.targets
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+***********************************************************************************************
+-->
+<Project>
+  <PropertyGroup>
+    <AndroidNETSdkVersion>@AndroidPackVersionLong@</AndroidNETSdkVersion>
+    <XamarinAndroidVersion>@AndroidPackVersionLong@</XamarinAndroidVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetPlatformVersion)' == '31.0' ">
```

Comment:
Is there a way this could check for `<= 32` then use 32? Trying to think of what this would look like over time as more API levels are added.

Reply:
I changed the initial value of the Runtime pack version property to the following:

```xml
    <_AndroidRuntimePackId Condition=" '$(_AndroidTargetingPackId)' &lt; '@ANDROID_LATEST_STABLE_API_LEVEL@' ">@ANDROID_LATEST_STABLE_API_LEVEL@</_AndroidRuntimePackId>
```